examples/hybrid/driver.jl

```diff
@@ -119,10 +120,41 @@ end
 
 if config.parsed_args["check_conservation"]
     FT = Spaces.undertype(axes(sol.u[end].c.ρ))
-    @test sum(sol.u[1].c.ρ) ≈ sum(sol.u[end].c.ρ) rtol = 50 * eps(FT)
-    @test sum(sol.u[1].c.ρe_tot) +
-          (p.net_energy_flux_sfc[][] - p.net_energy_flux_toa[][]) ≈
-          sum(sol.u[end].c.ρe_tot) rtol = 100 * eps(FT)
+
+    # energy
+    energy_total = sum(sol.u[end].c.ρe_tot) - p.net_energy_flux_toa[][]
+    energy_atmos_change = sum(sol.u[end].c.ρe_tot) - sum(sol.u[1].c.ρe_tot)
+    sfc = p.atmos.surface_model
+    if sfc isa CA.PrognosticSurfaceTemperature
+        sfc_cρh = sfc.ρ_ocean * sfc.cp_ocean * sfc.depth_ocean
+        energy_total +=
+            horizontal_integral_at_boundary(sol.u[end].sfc.T .* sfc_cρh)
+        energy_surface_change =
+            horizontal_integral_at_boundary(
+                sol.u[end].sfc.T .- sol.u[1].sfc.T,
+            ) * sfc_cρh
+    else
+        energy_surface_change = 0
+    end
+    energy_radiation_input = -p.net_energy_flux_toa[][]
+    @test (energy_atmos_change + energy_surface_change) / energy_total ≈
+          energy_radiation_input / energy_total atol = sqrt(eps(FT))
+
+    if p.atmos.moisture_model isa DryModel
+        # density
+        @test sum(sol.u[1].c.ρ) ≈ sum(sol.u[end].c.ρ) rtol = 50 * eps(FT)
+    else
+        # water (some error due to not accumulating P at every substage)
+        water_total =
+            sum(sol.u[end].c.ρq_tot) .-
+            horizontal_integral_at_boundary(sol.u[end].sfc.P_minus_E)
```

Comment:
```suggestion
            sum(sol.u[end].c.ρq_tot)
```
(Same as the energy, I think it's slightly more clear if the total water is just defined as the total moisture in the atmosphere.)

Reply:
The `P_minus_E` is the amount of water stored at the surface (rather than a source like `net_energy_flux_toa` was above), so I think we want this to be included, so the sum reflects the total water in the atmosphere-surface system.

Reply:
Maybe `sfc.P_minus_E` should be renamed to `sfc.water` to clarify this, since it's actually `(P-E) * dt` after it's been integrated by the timestepper.

Reply:
Oh, I see. It might be good to keep the name sfc.P_minus_E then, to make it clear that it is calculated from P - E, rather than a prognostic variable of the surface (or slab ocean).

Reply:
Oh, I didn't realize you change the prognostic variable name to be `sfc.water` too. Then yes, I think it's more clear now.